src/appMain/init_policy.sh

```diff
@@ -1,6 +1,24 @@
-# This script star QDB server for SDL
+# This ksh script starts QDB server for SDL on QNX
 # Need superuser to start qdb
 
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/qnx650/target/qnx6/x86/usr/lib
-export LD_LIBRARY_PATH
-/usr/sbin/qdb -c policy.ini
+version=`uname -r`
+echo "QNX ${version}"
```

Comment:
@bonewell, looks liek it shall be build only on QNX system?
Whats about cross-building?